components/app/myrw/widgets/pages/edit.js

```diff
@@ -95,49 +95,56 @@ class WidgetsEdit extends React.Component {
 
     this.setState({ loading: true });
 
-    const { widget } = this.state;
+
+
+    const { widget, name } = this.state;
     const widgetAtts = widget.attributes;
-    const dataset = widgetAtts.dataset;
+    const { dataset } = widgetAtts;
     const { user } = this.props;
     const widgetConfig = (this.onGetWidgetConfig) ? await this.getWidgetConfig() : {};
 
+    const metadata = {
+      language: this.props.locale,
```

Comment:
@pablopareja see there is a task here: https://www.pivotaltracker.com/story/show/156874971 

Should I always use `en`?

Reply:
solved ^^